Sources/KlaviyoCore/Models/ObjectStateError.swift

```
@@ -0,0 +1,10 @@
+//
```

Same comment as the other PR - Do we anticipate this being used in other modules if not should be keep it in forms?

---

same comment as [here](https://github.com/klaviyo/klaviyo-swift-sdk/pull/329#discussion_r2039926730). This is a common error that's really not specific to KlaviyoForms; there are many parts of the code where we may want to throw an error if an expected object doesn't exist. But I can move it if you prefer.